services/src/handlers/wms.rs

```diff
@@ -365,14 +371,38 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn get_capabilities() {
+    async fn test_invalid_method() {
+        let res = test_test_helper("POST", None).await;
+
+        ErrorResponse::assert(&res, 405, "MethodNotAllowed", "HTTP method not allowed.");
+    }
+
+    #[tokio::test]
+    async fn test_missing_fields() {
+        let res = test_test_helper("GET", Some("/wms?service=WMS&version=1.3.0&layers=mock_raster&bbox=1,2,3,4&width=100&height=100&crs=foo&styles=ssss&format=image/png")).await;
```

Comment:
Can you please add a test with invalid field value, e.g. a string for the parameter width? And please try to give more detailed error message than 
> Unable to parse query string: could not parse string

which is returned currently

Reply:
a9528bb